packages/api-tests/test/window.spec.js

```diff
@@ -54,16 +54,328 @@ describe('Window API', function(done) {
     return executeAsyncJavascript(app.client, script);
   });
 
+  it('Should close the window', function() {
+    const windowTitle = 'windownameclose';
+    const windowOptions = {
```

Comment:
Could have a global `defaultWindowOptions` and just override what we need for each test?